api/notd/collection_processor.py

```diff
@@ -91,7 +108,7 @@ async def retrieve_collection(self, address: str) -> RetrievedCollection:
             instagramUsername=openseaCollection.get('instagram_username'),
             wikiUrl=openseaCollection.get('wiki_url'),
             openseaSlug=openseaCollection.get('slug'),
-            url=openseaCollection.get('external_url'),
+            url=openseaCollection.get('external_url') or contractMetadataUri,
```

Comment:
this isnt right - contractMetadataUri is not equivalent to the external url. the external url is usually a marketing website like creepz.co